frontend/src/components/pages/GlobalForms/GlobalFormsPage.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import {Text, Container} from '@chakra-ui/react'
+import AdminAPIClient from "../../../APIClients/AdminAPIClient";
+import { ClauseInfo } from "../../../types/AdminTypes";
+import WaiversTab from "./WaiversTab";
+
+const GlobalFormsPage = (): React.ReactElement => {
+
+  const [waiver, setWaiver] = React.useState([]as ClauseInfo[]);
+  React.useEffect(() => {
+    const getWaiver = async () => {
+      const res = await AdminAPIClient.getWaiver();
+      if (res.clauses.length !== undefined) setWaiver(res.clauses);
+    };
+
+    getWaiver();
+  }, []);
+
+  return (
+    <Container
```

Comment:
Can you run the following 2 commands to make sure you lint your code before merging? Linting is a way to auto-format our code based on the agreed formats we set up at the start of the project. Run these 2 commands whilst you have your Docker containers running:

`docker exec -it scv2_ts_backend /bin/bash -c "yarn fix"`

`docker exec -it scv2_frontend /bin/bash -c "yarn fix"`